PayPalWebPayments/src/main/java/com/paypal/android/paypalwebpayments/PayPalWebCheckoutClient.kt

```diff
@@ -98,6 +99,9 @@ class PayPalWebCheckoutClient internal constructor(
             is PayPalWebStatus.VaultSuccess -> notifyVaultSuccess(status.result)
             is PayPalWebStatus.VaultError -> notifyVaultFailure(status.error)
             PayPalWebStatus.VaultCanceled -> notifyVaultCancelation()
+            is PayPalWebStatus.UnknownError -> {
+                Log.d("PayPalSDK", "An unknown error occurred: ${status.error.message}")
```

Comment:
Same issue as seen [here in CardClient](https://github.com/paypal/paypal-android/pull/281/files#r1830010584).